Doc/issue 280 file format (#283)
AlphaGenes/AlphaPeel: docs/source/file_formats/data_file_formats/phenotype.rst

.. _pheno_file_format:

Phenotype
=========

Format
~~~~~~

The first value in each line is the individual's ID.
The remaining values are the *phenotypes* of the individual for a specific trait,
coded from ``0`` onwards.
For example, a binary trait should be coded as ``0`` and ``1``,
while a trait with three states should be coded as ``0``, ``1``, and ``2``
(current phenotype functionality works only with one locus and one trait).

Example with four individuals and their phenotypes for a binary trait:

::

  id1 0
  id2 1
  id3 1
  id4 0

Input details
~~~~~~~~~~~~~

This file has one or multiple lines with
*observed phenotypes* for each phenotyped individual.

> If provided, values from this file are translated to genotype probabilities. 

Reviewer: Move this to the end of this input (because there is mention of multiple lines etc matching with the above two lines) and tweak the text to match the others (see genotype input for example on "translation").

Multiple lines per individual support repeated phenotyping.

> The file does not need to include all individuals present in other files.

Reviewer: Add another line mentioning that non-phenotyped individuals should not be present in the file and we don't use 9 for missing phenotype (right @RosCraddock?)

Reviewer 2: That's correct, we do not use 9 for missing phenotypes. Any phenotypes missing are not to be included in the file.

.. _pheno_file_format:

Phenotype
=========

Format
~~~~~~

The first value in each line is the individual's ID.
The remaining values are the *phenotypes* of the individual for a specific trait,
coded from ``0`` onwards.
For example, a binary trait should be coded as ``0`` and ``1``,

> while a trait with three states should be coded as ``0``, ``1``, and ``2``

Reviewer: ```suggestion
while a trait with three states should be coded as ``0``, ``1``, and ``2``.
```

> (current phenotype functionality works only with one locus and one trait).

Reviewer: ```suggestion
Current phenotype functionality works only with one locus and one trait!
```

Example with four individuals and their phenotypes for a binary trait:

::

  id1 0
  id2 1
  id3 1
  id4 0

Input details
~~~~~~~~~~~~~

This file has one or multiple lines with
*observed phenotypes* for each phenotyped individual.
Multiple lines per individual support repeated phenotyping.
The file does not need to include all individuals present in other files.

> Non-phenotyped individuals should not be present in the file and 

Reviewer: ```suggestion
Non-phenotyped individuals should not be present in the file;
that is, missing phenotypes are not to be included in the file.
```

> we don't use 9 for missing phenotype. 

Reviewer: remove this line

> Any phenotypes missing are not to be included in the file.

Reviewer: remove this line

If provided, values from this file are translated to :ref:`penetrance <penetrance>` 

> component of genotype probabilities (see :ref:`genotype input <genotype_input>` for example on "translation"). 

Reviewer: ```suggestion
component of genotype probabilities. 
```

This file has one or multiple lines with
*observed phenotypes* for each phenotyped individual.
Multiple lines per individual support repeated phenotyping.
The file does not need to include all individuals present in other files.
Non-phenotyped individuals should not be present in the file and 
we don't use 9 for missing phenotype. 
Any phenotypes missing are not to be included in the file.

> If provided, values from this file are translated to genotype probabilities

Reviewer: But, isn’t this wrong!? Phenotype only informs the “phenotype penetrance” term, which is just one of multiple terms (in addition to “prior” from parents and “posterior” from progeny) that give us genotype probabilities? Similarly observed genotypes inform only on the “genotype penetrance” term. So, it’s a bit more nuanced than “translate to genotype probs”.
